go/vt/vtctl/reparent.go

```diff
@@ -18,12 +18,15 @@ import (
 )
 
 var (
-	disableActiveReparents = flag.Bool("disable_active_reparents", false, "if set, do not allow active reparents. Use this to protect a cluster using external reparents.")
+	// DisableActiveReparents is a flag to disable active
+	// reparents for safety reasons. It is exported so the UI
+	// menus can be disabled.
```

Comment:
Yes, the first sentence says we disable active reparents. The second sentence says it is exported so the UI can see it. I'll make it clearer.